examples/Spar/Server/ClientApp/.gitignore

```diff
@@ -0,0 +1,40 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
Why is there a .gitignore in here? Can it be moved to the root one instead?

Reply:
A nested gitignore is a standard part of the template. I think it is fine.